Update loadbalancing.md (#29226)
dotnet/AspNetCore.Docs: aspnetcore/grpc/loadbalancing.md

The preceding code:

* Configures the created channel with the address `dns:///my-example-host`.
  * The `dns` scheme maps to `DnsResolverFactory`.
  * `my-example-host` is the hostname or IP address to resolve.

>   * A port can optionally be specified after the DNS address. For example `dns:///my-example-host:8080` configures gRPC calls to be sent to port 8080. If no port is specified then gRPC calls are sent to the default port.

Author: ```suggestion
  * No port is specified so gRPC calls are sent to the default port. A port can optionally be specified after the DNS address. For example `dns:///my-example-host:8080` configures gRPC calls to be sent to port 8080.
```

> * Configures the created channel with the address `dns:///my-example-host`.
>   * The `dns` scheme maps to `DnsResolverFactory`.
>   * `my-example-host` is the hostname or IP address to resolve.
>   * No port is specified in the address so gRPC calls are sent to the default port. A port can optionally be specified after the DNS hostname or IP address. For example, `dns:///my-example-host:8080` configures gRPC calls to be sent to port 8080.

Author: ```suggestion
* Configures the created channel with the address `dns:///my-example-host`.
  * The `dns` scheme maps to `DnsResolverFactory`.
  * `my-example-host` is the hostname to resolve.
  * No port is specified in the address, so gRPC calls are sent to port 80. This is the default port for unsecured channels. A port can optionally be specified after the hostname. For example, `dns:///my-example-host:8080` configures gRPC calls to be sent to port 8080.
```
